.agents/manifest.md

```
@@ -0,0 +1,60 @@
+<!--
+Source: Based on Obsidian community guidelines
+Last synced: See sync-status.json for authoritative sync dates
+Update frequency: Check Obsidian releases repo for validation requirements
+-->
+
+# Manifest rules (`manifest.json`)
+
+## Required Fields
+
+All plugins must include these fields in `manifest.json`:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)